client/state/selectors/are-all-sites-single-user.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import { isSingleUserSite } from 'state/sites/selectors';
+
+/**
+ * Returns true if every site of the current user is a single user site
+ *
+ * @param  {Object}  state Global state tree
+ * @return {Boolean}       True if all sites are single user sites
+ */
+export default function areAllSitesSingleUser( state ) {
```

Comment:
Might be worth memoizing with [createSelector](https://github.com/Automattic/wp-calypso/tree/master/client/lib/create-selector#create-selector)